packages/admin/admin/src/form/FinalFormFileSelect.tsx

```diff
@@ -0,0 +1,273 @@
+}) => {
+    const { delete: deleteIcon = <Delete />, info: infoIcon = <Error color="error" />, select: selectIcon = <Select /> } = iconMapping;
+
+    const onDrop = React.useCallback(
+        (acceptedFiles: File[]) => {
+            if (multipleFiles) {
+                if (Array.isArray(fieldValue)) {
+                    if (!maxFiles || (maxFiles && fieldValue.length < maxFiles && fieldValue.length + acceptedFiles.length <= maxFiles)) {
+                        onChange([...fieldValue, ...acceptedFiles]);
+                    }
+                } else {
+                    onChange([...acceptedFiles]);
+                }
+            } else {
+                onChange(acceptedFiles[0]);
+            }
+        },
+        [fieldValue, multipleFiles, onChange, maxFiles],
+    );
+
+    const removeFile = (removedFile: File) => () => {
+        const newFiles = Array.isArray(fieldValue) ? fieldValue.filter((file) => file !== removedFile) : undefined;
+        onChange(newFiles);
+    };
+
+    const { fileRejections, getRootProps, getInputProps, isDragReject } = useDropzone({
+        onDrop,
+        accept,
+        disabled: disabled || (maxFiles && fieldValue.length >= maxFiles),
+        multiple: multipleFiles,
+        maxSize: maxSize,
+        maxFiles,
+    });
+
+    let acceptedFiles: File[] = [];
+
+    if (Array.isArray(fieldValue)) {
+        acceptedFiles = fieldValue;
+    } else if (fieldValue.name !== undefined) {
+        acceptedFiles = [fieldValue];
+    }
+
+    const rejectedFiles = fileRejections.map((rejectedFile, index) => (
+        <div key={index} className={classes.rejectedFileListItem}>
+            <Tooltip trigger="hover" title={rejectedFile.file.name}>
+                <div className={classes.fileListText}>{rejectedFile.file.name}</div>
+            </Tooltip>
+            {infoIcon}
+        </div>
+    ));
+
+    return (
+        <div className={classes.root}>
+            <div {...getRootProps()} className={classes.dropzone}>
+                <input {...getInputProps()} />
+                {!disableDropzone && (
+                    <div
+                        className={clsx(
+                            classes.droppableArea,
+                            disabled && classes.droppableAreaIsDisabled,
+                            isDragReject && classes.droppableAreaHasError,
+                        )}
+                    >
+                        {isDragReject && <div className={classes.droppableAreaError}>{infoIcon}</div>}
+                        <Typography variant="body2" className={classes.droppableAreaCaption}>
+                            <FormattedMessage id="comet.finalFormFileSelect.dropfiles" defaultMessage="Drop files here to upload" />
+                        </Typography>
+                    </div>
+                )}
+                {!disableSelectFileButton && (
+                    <Button disabled={disabled} variant="contained" color="secondary" startIcon={selectIcon}>
```

Comment:
This button is green, but should be dark grey, according to the design. 

@thomasdax98 will the dark grey button be added to `next` anytime soon? 
Then we could change the base of this PR to `next` unless we need this in V6. 
Or should we add custom styles for this button here?

<img width="729" alt="file upload design" src="https://github.com/vivid-planet/comet/assets/6264317/5817cd2d-d4a1-48bd-9a22-7acfc9960f8d">

Reply:
We need this in v6 too, so we can't change the base to next.

But it should definitely not be green. According to Figma, it should be `grey[800]`: 

<img width="147" alt="Bildschirmfoto 2024-05-02 um 10 03 32" src="https://github.com/vivid-planet/comet/assets/13380047/ffb35f71-8da1-49cb-9ef7-d7c16d67bae4">

So let's use `grey[800]` from the theme. It's slightly off in main, but in next it's correct (`#333333`). To me this slight divergence in main is acceptable.